CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ Changes to Calva.
 ## [Unreleased]
 
 - Fix [Suppling a custom printFn to the pretty printer does not work](https://github.com/BetterThanTomorrow/calva/issues/1979)
+- Fix [Running a single test runs all tests (polylith)](https://github.com/BetterThanTomorrow/calva/issues/1981)
```

Comment:
Does this only affect polylith projects? It seems it would affect any project.

Reply:
Yes it affects all projects. I was quoting the name of the issue, I'll change it now.

Reply:
Thanks! Please consider changing the name of the issue as well.